examples/integration_test.py

```diff
@@ -890,6 +891,69 @@ def stats_cb(stats_json_str):
     c.close()
 
 
+class SerializerTest(Serializer):
+    def serialize(self, topic, value):
+        return value[::-1]
+
+
+class DeserializerTest(Deserializer):
+    def deserialize(self, topic, value):
+        return value[::-1]
+
+
+def verify_serializer():
+    conf = {'bootstrap.servers': bootstrap_servers,
+            'api.version.request': api_version_request,
+            'max.in.flight.requests.per.connection': 1,
+            'error_cb': error_cb,
+            'acks': -1
+            }
+
+    t = "verify_serializer_{}".format(uuid.uuid4())
+
+    p = confluent_kafka.Producer(conf, key_serializer=SerializerTest())
```

Comment:
It seems like this serializer does not modify the data, does not actually serialize, so even if it fails to work the tests will pass.
Use something ubiquitous like  base64